tad/api/routes/status.py

```diff
@@ -0,0 +1,11 @@
+from fastapi import APIRouter
+from fastapi.responses import JSONResponse
+
+from tad.core.config import VERSION
+
+router = APIRouter()
+
+
+@router.get("/version", response_class=JSONResponse)
```

Comment:
Why not return the version in the health respons?

Reply:
Though about that and figured maybe that wasn't the right place for it. But I'm ok with putting it there.